internal/cli/server/config.go

```diff
@@ -904,6 +902,61 @@ var (
 	gitDate          = "" // Git commit date YYYYMMDD of the release (set via linker flags)
 )
 
+// tries unlocking the specified account a few times.
+func unlockAccount(ks *keystore.KeyStore, address string, i int, passwords []string) (accounts.Account, string) {
```

Comment:
was this func present earlier?

Reply:
Yes it is present in main package. Cannot call it directly due to cyclic dependency.